src/annotator/sidebar.js

```diff
@@ -225,11 +244,36 @@ export default class Sidebar {
    * @param {MessagePort} port
    */
   onFrameConnected(source, port) {
-    if (source === 'sidebar') {
-      this._sidebarRPC.createChannel(port);
+    switch (source) {
+      case 'guest':
+        this._guestRPC.createChannel(port);
+        break;
+      case 'sidebar':
+        this._sidebarRPC.createChannel(port);
+        break;
     }
   }
 
+  _setupGuestEvents() {
+    this._guestRPC.on(
+      'textSelectedAt',
+      /** @type {string} */ frameIdentifier => {
```

Comment:
See comments about `@type` in guest.js. You'll need to use `@param` here I think.